fetools-app/src/components/CharacterFinderComponents/DisplaySearchResults.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import CharacterCard from "./CharacterCard";
+import htmlCharacters from "./htmlCharacters.json";
+
+function DisplaySearchResults({ searchQuery }) {
+  const allCharacters = Object.values(htmlCharacters);
+
+  const filteredCharacters = searchQuery
+    ? allCharacters.filter((character) => {
+        const searchString = searchQuery.toLowerCase();
+        return (
+          character.name.toLowerCase().includes(searchString) ||
+          character.character.toLowerCase().includes(searchString) ||
+          character.hex.toLowerCase().includes(searchString) ||
```

Comment:
Here we'd need to narrow the search down a bit:
* Only search character.hex if the search query includes '&#' and at least one more character.
* Don't search character.dec.
* Only search character.css if the search query includes '\' and at least one more character.